### PR TITLE
Updated plots

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Update plots to match the paper Figure 2.

- Treatment labels
- Normalisation
- I also added an inverted copy of the treatments in Figure 2, to match the paper figure

The grouping is unchanged at present.

Compare with [submitted figure](https://drive.google.com/drive/folders/0BwwQLcImmFJSNXpScDFYT2xtY2M).

/cc @sbesson

### CODE REVIEW

File: analysis/summary-metadata-repeated.tsv [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-"Java/Windows"	"tubhiswt"	"metadata"	"read"	48.45	118.364501701979	2603	7805	177	215	10408	2.44302377093869	214.819401444788	524.808904188439	3.56762770897833	8.71579929889359
-"Java/Windows"	"tubhiswt"	"metadata"	"write"	132.7	276.214847727748	2603	7805	177	215	10408	2.08149847571777	78.4325546345139	163.257242918392	1.3025739449887	2.71130568100366
-"Java/Windows"	"BBBC"	"metadata"	"read"	227.75	178.875397678766	37088	93001	2368	3456	130089	0.785402404736625	571.192096597146	448.615646233953	10.1536772777168	7.9747225508384
-"Java/Windows"	"BBBC"	"metadata"	"write"	1381.9	407.72422310339	37088	93001	2368	3456	130089	0.295046112673414	94.13778131558	27.7749864328618	1.67342065272451	0.493736258453773
-"Java/Windows"	"MitoCheck"	"metadata"	"read"	335.9	195.946260087059	55363	179881	3237	122	235244	0.583347008297287	700.339386722239	408.540886037175	9.41093424382257	5.48984033641639
-"Java/Windows"	"MitoCheck"	"metadata"	"write"	4725.8	506.056643386676	55363	179881	3237	122	235244	0.10708380451705	49.7786618138728	5.33048849079711	0.668909562931144	0.0716293808765042
+"C++/Linux"	"5D"	"metadata"	"read"	102	42.1276256404899	NA	NA	NA	NA	NA	0.413015937651862	NA	NA	NA	NA
```
Comment (sbesson) [inline analysis/summary-metadata-repeated.tsv]: Why are some of the metrics marked as `NA` ?

**Review by sbesson — COMMENTED** [review]
Overall looks good and accurately ports the latest figure calculations to the source code repository.

Two comments:
- as we tagged `v0.1.0` and already started modifying `master` for the ongoing tiling work, I will derive a stable `0_1` or `benchmark001` branch and modify the base branch of this PR. Once all modifications have been included, we should be able to tag this as `v0.1.1` and open a PR to merge this tag into `master`.
- in terms of naming, :+1: for generating multiple versions of the relative performance figures although the fact `files-fig2-inverted.pdf` matches Figure 2 might be confusing.

With respect to the latter point, the simplest way to resolve this would be to rename `files-fig2.pdf` as `files-fig2-inverted.pdf` and vice-versa. Although as `suppfigxx` are no longer relevant, an alternative would be to explicitly name these figures e.g. `relative_performance_by_implementation.pdf`. We could expand the last section of the README to list the various figures, indicate the correspondence with the paper but also refer to them while discussing conclusions like the JVM optimization.

File: analysis/summary-metadata-repeated.tsv (hunk 1 above)
Comment (rleigh-codelibre) [reply analysis/summary-metadata-repeated.tsv]: Good question, looks like those columns have somehow been dropped, though it's certainly not intentional.  I'll double-check this.

Comment (rleigh-codelibre) [reply analysis/summary-metadata-repeated.tsv]: It's due to renaming the datasets to match the paper; fixed now.

**Comment by rleigh-codelibre** [comment]
I've renamed the figures so Figure 2 is the paper version, and what was Figure 2 is now Supplementary Figure 1, with the numbers of all the other supplementary figures being incremented by one.  Is this all OK?

**Comment by sbesson** [comment]
All looks good now thanks. :+1: for matching the Figure 2 and the a set of incremental supplementary figures makes complete sense to me. I tried  switching the base branch of this PR but as it is based on `master` it also included the cleanup changes. Merging as it is and I will rebase onto the `0_1` branch.

**Comment by sbesson** [comment]
--rebased-to #37

**Pull request merged**